src/benchmarks/pmembench.c

```diff
@@ -891,23 +873,26 @@ static int
 pmembench_remove_file(const char *path)
```

Comment:
http://en.cppreference.com/w/cpp/experimental/fs

Or use a different file system library. Or write one. But don't do this.